etna/core/models/settings.py

```diff
@@ -12,15 +13,23 @@
 
 @register_setting(icon="list-ul")
 class SiteSettings(BaseSetting, ClusterableModel):
-    panels = [
-        InlinePanel(
-            "main_menu_items_rel",
-            heading="main menu items",
-            label="item",
-            min_num=1,
-            max_num=6,
-        ),
-    ]
+    stand_first = models.CharField(max_length=200, blank=False, null=True)
+    banner_link = models.URLField(blank=True, null=True, help_text="Banner URL")
+    banner_text = RichTextField(blank=True, null=True)
+    panels = (
+        [
+            InlinePanel(
+                "main_menu_items_rel",
+                heading="main menu items",
+                label="item",
+                min_num=1,
+                max_num=6,
+            ),
+        ]
+        + [FieldPanel("stand_first")]
+        + [FieldPanel("banner_link")]
+        + [FieldPanel("banner_text")]
```

Comment:
I'm a little bit confused about the joining of multiple lists here. Usually panels would just be a single list. Generally speaking, it usually makes for a better interface to have fixed fields first, with inline panels at the end (as they hog quite a bit of screen real-estate). We can also use `MultiFieldPanel` to better group the fields together, like so:

```suggestion
    panels = [
        MultiFieldPanel(
            heading="beta banner",
            children=[
                FieldPanel("beta_banner_standfirst"),
                FieldPanel("beta_banner_text"),
                FieldPanel("beta_banner_link"),
            ]
        ),
        InlinePanel(
            "main_menu_items_rel",
            heading="main menu items",
            label="item",
            min_num=1,
            max_num=6,
        ),
    ]
```